### PR TITLE
[Orchestrator] Enable explorer by default

### PR DESCRIPTION
**Pull request by JLineaweaver**
### What does this PR do?

- Enable Orchestrator Explorer by default. Both the Operator and Helm Charts do this by default. However, we want to make sure users have this turned on by default and let feature discovery handle turning on the checks.

### Additional Notes

- Removed an orchestrator_explorer disabled piece of testing. The reason being is this will always be enabled by default and rely on autodiscovery to not use the orchestrator checks.

### CODE REVIEW

**Review by jeremy-hanna — APPROVED** [review]

**Review by kangyili — APPROVED** [review]

File: pkg/config/setup/config.go [hunk 1]
```diff
@@ -744,7 +744,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.SetKnown("listeners")
 
 	// Orchestrator Explorer DCA and process-agent
```
[the remark this answers is not in the document]
Comment (JLineaweaver) [reply pkg/config/setup/config.go]: [e8ef2a8](https://github.com/DataDog/datadog-agent/pull/25027/commits/e8ef2a888ee0148244c5aed62b832d2befb740ca)

**Review by git-thuerk-done — APPROVED** [review]
good for docs

**Review by tobz — APPROVED** [review]

File: pkg/aggregator/no_orchestrator.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+//go:build !orchestrator
+
+package aggregator
+
+// Orchestrator Explorer is enabled by default but
+// the forwarder is only created if the orchestrator
+// build tag exists
+
+// OrchestratorForwarderSupport shows if the orchestrator build tag is enabled
+const OrchestratorForwarderSupport = false
```
Comment (vickenty) [inline pkg/aggregator/no_orchestrator.go:15]: Would it make sense to make this const private?

```suggestion
const orchestratorForwarderSupport = false
```

Comment (JLineaweaver) [reply pkg/aggregator/no_orchestrator.go]: [e08d6ce](https://github.com/DataDog/datadog-agent/pull/25027/commits/e08d6cea6c381a68caa8bb7ba0ea8b47e1349dce)

**Review by vickenty — APPROVED** [review]

**Pull request merged**